project/Dependencies.scala

```diff
@@ -22,9 +22,9 @@ object Dependencies {
     val postgres = "42.1.4"
 
     val endpoints = "0.9.0"
-    val cats = "1.6.0"
+    val cats = "2.1.1"
```

Comment:
what's included in new cats that we use here?

Reply:
The Monocle latest is based on that

Reply:
Or, to be precise, it's based on cats 2.x